### DISCUSSION TITLE
Code not Working?! Please help!

### DISCUSSION THREAD

**Discussion by saucek26** [kyleskom/NBA-Machine-Learning-Sports-Betting#116] [Q&A]
fixed it

**Comment by nkgilley** [comment]
duplicate of https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/issues/108 & https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/issues/111 & https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/issues/113. fixed in https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/pull/109
